examples/dt/uni04delta/edpm-pre-ceph.md

```diff
@@ -0,0 +1,58 @@
+# Configuring and deploying Ceph nodes
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been created and successfully
+  deployed.
+
+## Steps
+
+```bash
+# 1. Switch to openstack namespace
+
+    $ oc project openstack
+
+# 2. Change the working directory to uni04delta/edpm-pre-ceph/nodeset
+
+    $ pushd architecture/examples/dt/uni04delta/edpm-pre-ceph/nodeset
+
+# 3. Modify [values.yaml](values.yaml) file to suit your environment.
+
+    $ vi values.yaml
+
+# 4. Generate the Ceph nodeset deployment plan.
+
+    $ kustomize build --load-restrictor LoadRestrictionsNone > nodeset-pre-ceph.yaml
+
+# 5. Create the CRs
+
+    $ oc apply -f nodeset-pre-ceph.yaml
+
+# 6. Wait for Ceph nodeset deployment to complete
+
+    $ oc wait osdpns ceph-nodes --for condition=SetupReady --timeout=600s
+    $ popd
+
+# 7. Change the working directory to uni04delta/edpm-pre-ceph
+
+    $ pushd architecture/examples/dt/uni04delta/edpm-pre-ceph
+
+# 8. Generate the Ceph data plane deployment plan.
+
+    $ kustomize build --load-restrictor LoadRestrictionsNone > deployment-pre-ceph.yaml
```

Comment:
Please remove `--load-restrictor LoadRestrictionsNone`. I was able to build `deployment-pre-ceph.yaml` without it.

Reply:
Done